components/dropdown/src/auro-dropdown.js

```diff
@@ -22,16 +21,11 @@ import styleCss from "./styles/style-css.js";
 import colorCss from "./styles/color-css.js";
```

Comment:
iconVersion should be from `root/src/iconVerion.js` and `components/dropdown/src/iconVersion.js` should be removed